ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -72,6 +72,14 @@
   public static final String
       KSQL_TABLE_STATESTORE_NAME_SUFFIX_DEFAULT = "_ksql_statestore";
 
+  public static final String
+      KSQL_OUTPUT_TOPIC_NAME_PREFIX_CONFIG = "ksql.output.topic.name.prefix";
+  private static final String KSQL_OUTPUT_TOPIC_NAME_PREFIX_DOCS =
+      "A prefix to add to any output topic names, where the statement does not include an explicit "
+      + "topic name. E.g. given 'ksql.output.topic.name.prefix = thing', then statement "
```

Comment:
It doesn't look like the code doesn't add the `-`, but the example here has the prefix defined as `thing` and then an output topic `thing-S`. Should the example prefix be `thing-`?

Reply:
Yep.